src/Hit.cc

```diff
@@ -195,8 +195,7 @@ void Hit::fillModuleLocalResolutionStats() {
 void Hit::setHitPassiveElement(const insur::InactiveElement* myPassiveElem) {
 
   if (myPassiveElem) m_hitPassiveElem = myPassiveElem;
-  else m_hitPassiveElem = nullptr;
-  //else logWARNING("Hit::setHitPassiveElement -> can't set inactive element to given hit, pointer null!");
+  else logWARNING("Hit::setHitPassiveElement -> can't set inactive element to given hit, pointer null!");
```

Comment:
isn't maybe `m_hitPassiveElem = nullptr` missing here?

Reply:
I thought that `m_hitPassiveElem = nullptr` was introduced by Adinda to suppress warnings caused by an overlap between dead areas in the split-sensor geometry, which then anyway reappeared as an `ERROR` in [`Analyzer::fillRIPlotsPerMechanicalCategoryAndSubdetector`](https://github.com/tkLayout/tkLayout/blob/d5022c8ce69ff9c7bc041f27a5f351c75e855934/src/Analyzer.cc#L3752) down the chain. Now that the overlap is fixed, it's not relevant.
And the `nullptr` is anyway assigned in the `Hit` [initializer](https://github.com/tkLayout/tkLayout/blob/d5022c8ce69ff9c7bc041f27a5f351c75e855934/src/Hit.cc#L64C5-L64C21).